### ISSUE TITLE
ansible_local: bash: line 5: pip: command not found

### ISSUE THREAD

**Issue by dominikzogg** [hashicorp/vagrant#10950]
### Vagrant version

Vagrant 2.2.5

### Host operating system

Ubuntu 18.04
Ubuntu 19.04

### Guest operating system

bento/ubuntu-18.04

### Vagrantfile
```ruby
config.vm.box = 'bento/ubuntu-18.04'

config.vm.provision 'ansible_local' do |ansible|
  ansible.playbook = 'playbook.yml'
  ansible.install_mode = 'pip'
  ansible.version = '2.8.1'
  ansible.extra_vars = Object.new
  ansible.compatibility_mode = '2.0'
end
```

### Debug output

`bash: line 5: pip: command not found`

### Expected behavior
Pip should be successfully installed

### Actual behavior
Pip not found

PS: Downgrade to Vagrant 2.2.4 works.

**Comment by scuben** [comment]
I can confirm this issue.

**Comment by lamech** [comment]
Hi, we see this as well; see https://gitlab.com/aegir/aegir-dev-vm/issues/16 comparing Vagrant 2.2.4 to Vagrant 2.2.5 debug logs. The curl command to install pip in [the ansible_local provisioner](https://github.com/hashicorp/vagrant/blob/master/plugins/provisioners/ansible/cap/guest/pip/pip.rb) looks like it's still there, but isn't getting called by Vagrant anymore for some reason.

**Comment by Tanzy** [comment]
Yep, I just had this, reverted to Vagrant 2.2.4 and it worked without issues.

**Comment by hartey11** [comment]
Hi, how do you revert to vagrant 2.2.4?

**Comment by Tanzy** [comment]
Just uninstall and install the previous version. You may need to fix your plugins as well. There is a link on the download page to download the previous versions

**Comment by arafalov** [comment]
Looking at the release notes, the issue that may have introduced this is: #9584

**Comment by Troym** [comment]
Setting _ansible.pip_install_cmd_ in the ansible_local configuration also worked around the issue for me.

```
  config.vm.provision "ansible_local" do |ansible|
    ...
    ansible.pip_install_cmd = "curl https://bootstrap.pypa.io/get-pip.py | sudo python"
    ...
  end
```

**Comment by Raffaello** [comment]
i have done a fix for this issue here: https://github.com/hashicorp/vagrant/pull/11096
that could be merged and rolled forward into the ver 2.2.6. 

It would be nice to have a review on that. anyone here? thx. :)

**Comment by briancain** [comment]
Hey all! It looks like this was due to a typo when finalizing ansibles guest config. I've made a pull request to fix this: https://github.com/hashicorp/vagrant/pull/11098. It should be fixed in the next release. For now you can get around this by following the fix mentioned by @Troym . Thanks!
